samples/Bundling/parcel/js/index.js

```diff
@@ -1,32 +0,0 @@
-const { BlobServiceClient } = require("@azure/storage-blob");
```

Comment:
Why are you deleting all the repo-wide samples?

Reply:
It was a commit issue. Added them back, thanks